### PR TITLE
Release docs

### PR DESCRIPTION
**Pull request by qmx**

### CODE REVIEW

**Comment by qmx** [comment]
one thing that worries me (of which I still don't have an answer) is how we deal with quickstart versioning - should we have a fixed version for a quickstart too on the pom files? Currently they are being released as X.X.X-SNAPSHOT (which makes sense if we think that the user will fiddle using the quickstart as a base for his own changes).

wdyt?

File: RELEASE_HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+1. Tag
+    
+        git tag -s <release version> -m "Release <release version>"
+1. Stage the release
+        
+        ./release-utils.sh -r
+1. Reset version numbers to snapshots
+        
+        ./release-utils.sh -u -o <release version> -n <new snapshot version>
+1. Commit this
```
Comment (balunasj) [inline RELEASE_HOWTO.md:19]: I would just add a step here about team review, or test of the staged artifact.

File: RELEASE_HOWTO.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Quickstarts
+
+1. Tag
+
+        git tag -s <release version> -m "Release <release version>"
+1. Push this tag
+
+        git push aerogear <release version>
+1. Do a fresh clone of <https://github.com/jbossas/quickstart/>
```
Comment (balunasj) [inline RELEASE_HOWTO.md:30]: I would have a separate section on syncing the release with jboss-as/quickstarts.  Just to keep these two steps separate.

**Comment by balunasj** [comment]
For the release numbering for quickstarts - I think we should do it.  It looks better with the 1.0.0.MX tag is versions that way.

I think most developers that will use this as a base for updates would upgrade to the latest in repo, but update the version themselves.

**Pull request merged**